pyprocessing/renderer/tkinter/window.py

```diff
@@ -1,5 +1,5 @@
 from collections import deque
-from math import degrees
+from math import degrees, sin, cos
```

Comment:
why the change, sin and cos are not used in the file (according to flake8)

look at the CI output when it fails